solidity-frontend/src/main/java/org/sonarsource/solidity/frontend/Utils.java

```diff
@@ -26,6 +38,7 @@ public static SolidityParser returnParserUnitFromParsedFile(String file) {
     SolidityParser parser = new SolidityParser(tokenStream);
     tokenStream.fill();
     parser.addCommentsToSet(tokenStream.getTokens());
+    // parser.emptyLines = Utils.emptyLines(newFile);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This block of commented-out lines of code should be removed. ([squid:CommentedOutCodeLine](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3ACommentedOutCodeLine))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=21&issues=AWQn3a70YwZU9vvrp9dE&open=AWQn3a70YwZU9vvrp9dE)

[ISSUE_KEY]: AWQn3a70YwZU9vvrp9dE